tensorrt_llm/_torch/models/modeling_qwen3vl.py

```
@@ -350,6 +351,84 @@ def __call__(
+
+        if len(mm_handles) != 1:
+            # TODO: only support single multimodal item within a request for now
+            raise NotImplementedError("Only one mm_handle is supported for Qwen3 VL for now")
+
+        hidden_size = mm_handles[0]["tensor_size"][1]
+        num_deepstack_levels = len(self.config.vision_config.deepstack_visual_indexes)
+        expected_size = self.config.text_config.hidden_size * (1 + num_deepstack_levels)
+        if hidden_size != expected_size:
+            raise RuntimeError(
+                "Expected multimodal embedding to have hidden size {expected_size}, got {hidden_size}."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)